ngraph/frontend/frontend_manager/include/frontend_manager/frontend_exceptions.hpp

```diff
@@ -0,0 +1,122 @@
+// Copyright (C) 2021 Intel Corporation
```

Comment:
These changes are from PR #5993 , which are not in 'master' and not related to python bindings. You can skip review of this here and add comments to PR #5993 if needed

Reply:
PR #5993 is merged, this change is not appeared in PR